packages/salesforcedx-vscode-core/package.json

```diff
@@ -36,7 +36,7 @@
     "@salesforce/ts-types": "1.5.13",
     "adm-zip": "0.4.13",
     "applicationinsights": "1.0.7",
-    "glob": "^8.0.3",
+    "glob": "^7.1.2",
```

Comment:
This major version bump resulted in the windows int test failures (and identified a real issue so props to those tests). I attempted to fix it for a bit, but decided to go ahead and just revert the version bump for now.

Reply:
@gbockus-sf It looks like this was bumped in at least one other package as well as a dependabot PR.  Should it get rolled back in the other packages as well?  Looks like it was bumped in core, system-tests, and test-utils-vscode: https://github.com/forcedotcom/salesforcedx-vscode/pull/4388/files

Reply:
oh good catch.  I think we should go ahead with this PR as is and I'll revert those other instances separately

Reply:
So I went and reviewed the places we use glob and the reverted package is the only place where it's used in prod code verse tests.  The tests seem good with the bump so I think we're good to leave it as is and deal with just the core version bump (ticket created) https://github.com/forcedotcom/easywriter/issues/86

Reply:
Awesome. Thank you!